src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,149 @@
+.menu {
+  padding-top: 20px;
+  background-color: #3f5252;
+  box-sizing: border-box;
+  height: 100%;
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 30px;
+
+    @include onTablet {
+      margin-bottom: 55px;
+    }
+  
+    @include onDesktop {
+      margin-bottom: 45px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    color: #f1f5f4;
+
+    @include onTablet {
+      flex-direction: row;
+      gap: 123px;
+    }
+
+    @include onDesktop {
+      gap: 83px;
+    }
+  }
+
+  &__box {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 20px;
+
+    @include onTablet {
+      margin-bottom: 50px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 70px;
+    }
+
+    &--last {
+      margin-bottom: 40px;
+
+      @include onTablet {
+        margin-bottom: 50px;
+      }
+
+      @include onDesktop {
+        margin-bottom: 60px;
+      }
+    }
+  }
+
+  &__schedule {
+    font-size: 14px;
+    line-height: 140%;
+    color: #f1f5f4;
+  }
+
+  &__schedule-time {
+    font-weight: 600;
+  }
+
+  &__adress {
+    font-size: 14px;
+    line-height: 140%;
+  }
+
+  &__adress-link {
+    font-weight: 600;
+    color: #f1f5f4;
+  }
+
+  &__list {
+    list-style: none;
+  }
+
+  
+  &__item {
+    margin-bottom: 20px;
+
+    @include onTablet {
+      margin-bottom: 50px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 60px;
+    }
+
+    &:last-child {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__link {
+    font-family: "Playfair Display", sans-serif;
```

Comment:
fonts must be set for the whole page

Reply:
For the whole page i setted Raleway font. Playfair is for titles, this is an exclusion, that`s why i didn`t created an @extend.